### PR TITLE
Add popover component

### PR DESCRIPTION
**Pull request by styczynski**
This PR adds a new very basic popover component. (this is the replacement for #102)

```
<Popover
    content={
        <div>
            Hello! :))
        </div>
    }
>
    <button> Press me! </button>
</Popover>
```

image

### CODE REVIEW

File: frontend/src/components/universal/Popover/index.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.rap-popover-content {
+    z-index: 9999999 !important;
+}
```
Comment (mbalc) [inline frontend/src/components/universal/Popover/index.scss]: eeeeej no

File: frontend/src/components/universal/Popover/index.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.rap-popover-content {
+    z-index: 9999999 !important;
```
Comment (mbalc) [inline frontend/src/components/universal/Popover/index.scss]: I think this should be bigger than z-index of the container, not equal

File: frontend/src/components/universal/Popover/index.scss (hunk 1 above)
Comment (mbalc) [reply frontend/src/components/universal/Popover/index.scss]: (eof no newline)

File: frontend/src/components/universal/Popover/index.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+import * as ReactPopover from 'react-awesome-popover';
+import "react-awesome-popover/dest/react-awesome-popover.css";
+import "./index.scss";
+
+const PopoverTrigger = styled.div`
+`;
+
+const PopoverContent = styled.div`
+  background:white;
+  box-shadow: 0 0 0 1px rgba(16, 22, 26, 0.1), 0 2px 4px rgba(16, 22, 26, 0.2), 0 8px 24px rgba(16, 22, 26, 0.2);
```
Comment (mbalc) [inline frontend/src/components/universal/Popover/index.tsx]: what is going on here? maybe add a comment explaining why have three sets of `box-shadow` properties

File: frontend/src/components/universal/Popover/index.scss (hunk 1 above)
Comment (styczynski) [reply frontend/src/components/universal/Popover/index.scss]: Fixed.

File: frontend/src/components/universal/Popover/index.scss (hunk 2 above)
Comment (styczynski) [reply frontend/src/components/universal/Popover/index.scss]: The z-index in the `.tsx` file was not needed as the tooltip wrapper (i.e `.rap-popover-content` must have an absolute position and should be rendered with correct z-index - its content doesn't have to set that). That's why I removed completely z-index from `.tsx` file.

File: frontend/src/components/universal/Popover/index.tsx (hunk 3 above)
Comment (styczynski) [reply frontend/src/components/universal/Popover/index.tsx]: This is just the default style for `react-awesome-popover`. 💯

Comment (styczynski) [reply frontend/src/components/universal/Popover/index.tsx]: Anyway I added a comment on that

**Pull request merged**